recipes/regiis.rb

```diff
@@ -28,4 +28,5 @@
   command "%WINDIR%\\Microsoft.Net\\Framework64\\v4.0.30319\\aspnet_regiis.exe -i -enable > #{guard_file}"
   creates guard_file
```

Comment:
I may be wrong, but seeing the name guard_file my thought was that it's intended to keep the command from running on every chef-client run. I assume it's meant to run once create the file and then not run again as long as the file exists. If my assumption is correct then this is not working as intended as it runs this command everytime chef-client runs even with the file in place.